mpifileutils.spec

```diff
@@ -249,6 +250,12 @@ done
 %endif
 
 %changelog
+* Tue Feb 22 2022 Dalton A. Bohning <daltonx.bohning@intel.com> - 0.11.1-1
+- Update to v0.11.1
+- Remove patch2 in favor of upstream d5f41b3
```

Comment:
This is no longer accurate right?

Reply:
Previously was `v0.11.0..bdc379b + patch2` (two patches)
Now is `v0.11.1..0837021` (one patch)